Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -459,19 +458,19 @@ public void ActivateInput()
             {
                 var actionMap = m_Actions.TryGetActionMap(m_DefaultActionMap);
                 if (actionMap != null)
+                {
                     actionMap.Enable();
+                    m_EnabledActionMap = actionMap;
+                }
                 else
                     Debug.LogError($"Cannot find action map '{m_DefaultActionMap}' in '{m_Actions}'", this);
             }
-
             m_InputActive = true;
         }
 
         public void PassivateInput()
         {
-            // Disable all enabled action maps.
-            if (m_Actions != null)
-                m_Actions.Disable();
+            m_EnabledActionMap?.Disable();
```

Comment:
Seems that we basically we enable two scenarios now:

1) You set a default action map, maybe switch around between a couple maps with `SwitchActions` but only ever use one action map at a time.
2) You set no default action map and handle enabling/disabling maps as needed entirely yourself.

Seems good and simple enough to me for what we have ATM. Could be we need something more eventually (especially WRT handling UI input).

Docs will need updating based on this change. They reflect the old behavior.

Reply:
Where are the docs for this?

Reply:
`Components.md` where it mentions "Default Action Map" and "SwichActions".